src/test/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseServiceTest.java

```diff
@@ -314,7 +314,7 @@ public void rejectionPartAdmitNoMediationShouldNotUpdateDirectionsQuestionnaireD
             .buildRejectionWithDirectionsQuestionnaire();
 
         final Claim claim = SampleClaim.builder()
-            .withFeatures(ImmutableList.of(LA_PILOT_FLAG, DQ_FLAG))
+            .withFeatures(ImmutableList.of(DQ_FLAG.getValue(), LA_PILOT_FLAG.getValue()))
```

Comment:
Features should ideally be a Set<ClaimFeature>. Though I understand that this may be an impactful change that we probably don't want to do for JDDO.